sympy/stats/drv.py

```diff
@@ -178,13 +178,17 @@ def restricted_domain(self, condition):
         return conditional_domain
 
     def probability(self, condition):
+        complement = isinstance(condition, Ne)
+        if complement:
+            condition = Eq(condition.args[0], condition.args[1])
```

Comment:
Is there a test for this logic in drv.py?

Reply:
The tests at [L54](https://github.com/sympy/sympy/blob/master/sympy/stats/tests/test_discrete_rv.py#L54) and [L48](https://github.com/sympy/sympy/blob/master/sympy/stats/tests/test_discrete_rv.py#L48) in test_discrete_rv.py test for  condition of the type `Ne`. That should be a test for this logic.